src/bouwdossiers/batch.py

```diff
@@ -98,9 +98,30 @@ def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  #
     Add wabo dossier to the the bouwdossier model. Structure of import is
     almost identical to pre_wabo xml to avoid confusion since only a few
     mappings are different and new fields are added.
+
+    Originally this was for importing wabo dossiers. But not it is also
+    used for prewabo dossiers that have the same  XML structure as the
+    wabo dossiers. This structure for files that originate from the 'tussenbestand'
+    and are not archived in de edepot.
     """
+
+    # The intern number can be something like sdz_prewabo_1274 or sdc_33
+    # If prewabo is present it is a prewabo dossier.
     dossier = x_dossier.get('intern_nummer')
-    stadsdeel, dossiernr = dossier.split('_')
+    m = re.match(r"([a-z]+)_(?:([a-z]+)_)?(\d+)", dossier)
+    if not m:
+        log.error(f"Invalid intern_nummer {dossier} in {file_path}")
+        return count, total_count
+
+    stadsdeel = m.group(1)
+    wabo_tag = m.group(2)
+    dossiernr = m.group(3)
+
+    if wabo_tag and wabo_tag == 'prewabo':
+        # prewabo key2 dossiers numbers can have the same values, for the same stadsdeel as existing
+        # prewabo dossiers imported from the edepot. Therefore we add  the letter p to the stadsdeel
+        # to make the combination unique.
+        stadsdeel += 'p'
```

Comment:
Ik mis de testcases voor deze changes

Reply:
Dat zal ik toevoegen in  een volgende PR omdat deze al is gemergd